cea/demand/demand_main.py

```diff
@@ -104,8 +104,7 @@ def calc_buildings_less_100m2(building_properties):
         return list_buildings_less_100m2
     list_buildings_less_100m2 = calc_buildings_less_100m2(building_properties)
     if list_buildings_less_100m2 != []:
-        print('!!!Warning!!!, the next list of buildings have less than 100m2 of net area, CEA does not handle well thses'
-              'buildings, it might be that CEA fails: These are the buildings%s' % list_buildings_less_100m2)
+        print('Warning!, the next list of buildings have less than 100 m2 of gross floor area, CEA might fail: %s' % list_buildings_less_100m2)
```

Comment:
@JIMENOFONSECA I'm pretty sure it's "The _following_ list". You use "next" a lot in your prose and every time I think it sounds a bit wrong. I'm going to fix it and merge. I just wanted to give you this feedback, since writing is important to your career.